terraform/lambdas.tf

```diff
@@ -85,3 +85,17 @@ module "digester_function" {
   tags          = var.tags
 }
 
+module "exif_function" {
+  depends_on    = [ aws_iam_role_policy_attachment.lambda_bucket_access ]
+  source        = "./modules/meadow_lambda"
+  name          = "exif"
+  description   = "Function to extract EXIF metadata from an S3 object"
+  role          = aws_iam_role.lambda_role.arn
+  stack_name    = var.stack_name
+  memory_size   = 1024
```

Comment:
@mbklein I copied the `memory_size` value from the digester function, but should we bump this up for processing large TIFFs?

Reply:
Yeah, this should probably be at least 4096 since you're reading the entire thing into memory.

Reply:
Bumped to 4096.